### ISSUE TITLE
Improvement: Community site - Use styled components instead of SCSS

### ISSUE THREAD

**Issue by VargaElod23** [Taraxa-project/taraxa-platform#91]
It would be great to use the same styling mechanism across the whole ecosystem, therefore it'd be great to use styled components not only in the Taraxa UI but in the community site too.

**Comment by farebord** [comment]
@rattrap We are not using `styled-components` in the Taraxa UI library. We are using MUI styles. Which are not the same.
We can use `styled-components namespace` to ensure we have uniques classes in both.
```
{
  "plugins": [
    ["babel-plugin-styled-components", {
      "namespace": "my-app"
    }]
  ]
}
```

**Comment by VargaElod23** [comment]
Yes, that should be better than what we have now, and later on fully convert to one styling system.

**Comment by farebord** [comment]
Sounds great! Thanks @VargaElod23
